source/_references/sourceimages.md

```diff
@@ -141,6 +141,20 @@ $isDeleted = $client->deleteSourceImagesWithBinaryHash('03b3e8a0bdd76ef55c021066
 var_dump($isDeleted);
 ```
 
+## Restore a source image
+
+To restore a deleted image,  do a POST on the `/sourceimages` route with providing the identifying hash and appending '/restore'.
+Returns a 200 http code and the meta data for the image, if the image could be restored. Also if the images wasn't deleted and didn't need to be restored.
+Returns a 404, if the image could not be restored or was not fonud.
+
+Images can be restored for about 30 days after their deletion.
```

Comment:
*about* 30 days?
On which factors does it depend on to not be 30 days?

Reply:
It's 30 days, may be more depending on the cleanup ;) But I just removed the about in the two places